waagent

```diff
@@ -5909,8 +5916,14 @@ def main():
         else:
             args.append(a)
     global Config
-    Config = ConfigurationProvider()
+    Config = ConfigurationProvider(conf_file)
     
+    logfile = Config.get("Logs.File")
+    if logfile is not None:
```

Comment:
The setting remains the same, only now the value isn't hardcoded, but explicitly provided by the config. 

I'm setting the default to stdout, because otherwise it's really hard to find where the log gets written without digging in this source file, which is, frankly, quite hard to do for a normal human being.